packages/lib-classifier/src/plugins/tasks/TranscriptionTask/models/TranscriptionTask.js

```diff
@@ -0,0 +1,31 @@
+import cuid from 'cuid'
+import { types } from 'mobx-state-tree'
+import Task from '../../models/Task'
+import DrawingTask from '../../DrawingTask/models/DrawingTask'
+import { TranscriptionLineTool } from '@plugins/drawingTools/models/tools'
+import { TranscriptionLine } from '@plugins/drawingTools/models/marks'
+import TranscriptionAnnotation from './TranscriptionAnnotation'
+
+// Similar to DrawingTask, but let's limit to only allowing 
+// TranscriptionLine tool, marks, and TranscriptionAnnotation models
+const Transcription = types.model('Transcription', {
+  activeMark: types.safeReference(TranscriptionLine),
+  annotation: types.safeReference(TranscriptionAnnotation),
+  caesarKey: types.maybe(types.literal('alice')),
```

Comment:
Maybe `types.frozen` here so that it can't be set to undefined? I'm not sure what the best way is to specify a constant value in MST, without requiring that it be passed in with every snapshot.

Reply:
Or maybe `types.optional(types.literal('alice'), 'alice')`.